policyengine_us/variables/gov/states/ct/tax/income/additions/ct_state_addback.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class ct_state_addback(Variable):
```

Comment:
could we point to the exact place in the references where this is outlined and could you compare to the add back file which we took as template (the sources) - I know I mentioned we need this but I am not 100% sure where I saw it anymore